### ISSUE TITLE
No valid response from Mobile Alerts server

### ISSUE THREAD

**Issue by MichaelSchaaf999** [bluerai/ioBroker.mobile-alerts#16]
Getting an error thrown:

mobile-alerts.0 | 2021-06-25 09:00:00.805 | error | No valid response from Mobile Alerts server
-- | -- | -- | --

What can we do about this?

Without a hint or a fix (if this is a bug) this adapter is useless.

BR Michael

**Comment by bluerai** [comment]
Hi Michael, still getting this error?

if yes, check the Adapter-Webpage from the instance-Tab in ioBroker. 
Does the page show your sensors overview? If not, check your configuration.

If your configuration is ok then set the mobile alerts adapter in debug mode and send me the error log entries.

bluerai

**Comment by MichaelSchaaf999** [comment]
Dear bluerai,

yes: The instance-Tab page does show my sensors overview.

Edited the instance and now debug/warn is shown.
“send me the error log entries”: From the IOBroker-log? Are u talking about messages like this:



2021-06-26 15:21:00.840

error

No valid response from Mobile Alerts server


***@***.***D76A9F.527320A0]
mobile-alerts.0

2021-06-26 15:21:00.554

debug

out: main

***@***.***D76A9F.527320A0]
mobile-alerts.0

2021-06-26 15:21:00.551

debug

https://measurements.mobile-alerts.eu/Home/SensorsOverview?phoneid=549301590686

***@***.***D76A9F.527320A0]
mobile-alerts.0

2021-06-26 15:21:00.551

debug

in: main v1.02

***@***.***D76A9F.527320A0]
mobile-alerts.0

2021-06-26 15:21:00.543

info

starting. Version 0.0.1 in /opt/iobroker/node_modules/iobroker.mobile-alerts, node: v12.22.1, js-controller: 3.3.14

***@***.***D76A9F.527320A0]
mobile-alerts.0

2021-06-26 15:21:00.428

debug

statesDB connected

***@***.***D76A9F.527320A0]
mobile-alerts.0

2021-06-26 15:21:00.427

debug

States connected to redis: 127.0.0.1:9000

***@***.***D76A9F.527320A0]
mobile-alerts.0

2021-06-26 15:21:00.383

debug

States create User PubSub Client

***@***.***D76A9F.527320A0]
mobile-alerts.0

2021-06-26 15:21:00.383

debug

States create System PubSub Client

***@***.***D76A9F.527320A0]
mobile-alerts.0

2021-06-26 15:21:00.379

debug

Redis States: Use Redis connection: 127.0.0.1:9000

***@***.***D76A9F.527320A0]
mobile-alerts.0

2021-06-26 15:21:00.379

debug

objectDB connected

***@***.***D76A9F.527320A0]
mobile-alerts.0

2021-06-26 15:21:00.376

debug

Objects connected to redis: 127.0.0.1:9001

***@***.***D76A9F.527320A0]
mobile-alerts.0

2021-06-26 15:21:00.368

debug

Objects client initialize lua scripts

***@***.***D76A9F.527320A0]
mobile-alerts.0

2021-06-26 15:21:00.367

debug

Objects create User PubSub Client

***@***.***D76A9F.527320A0]
mobile-alerts.0

2021-06-26 15:21:00.367

debug

Objects create System PubSub Client

***@***.***D76A9F.527320A0]
mobile-alerts.0

2021-06-26 15:21:00.366

debug

Objects client ready ... initialize now

***@***.***D76A9F.527320A0]
mobile-alerts.0

2021-06-26 15:21:00.352

debug

Redis Objects: Use Redis connection: 127.0.0.1:9001

***@***.***D76A9F.527320A0]
mobile-alerts.0

2021-06-26 15:17:00.482

error

Termination forced due to timeout!


Hope this is what you are talking about.

Best regards
Michael


Michael Schaaf  | fon +49 172 5233633  | fax +49  7172 9340030   | mail ***@***.******@***.***>


Von: bluerai ***@***.***>
Antworten an: "bluerai/ioBroker.mobile-alerts" ***@***.***>
Datum: Samstag, 26. Juni 2021 um 15:10
An: "bluerai/ioBroker.mobile-alerts" ***@***.***>
Cc: Michael Schaaf ***@***.***>, Author ***@***.***>
Betreff: Re: [bluerai/ioBroker.mobile-alerts] No valid response from Mobile Alerts server (#16)


Hi Michael, still getting this error?

if yes, check the Adapter-Webpage from the instance-Tab in ioBroker.
Does the page show your sensors overview? If not, check your configuration.

If your configuration is ok then set the mobile alerts adapter in debug mode and send me the error log entries.

bluerai

**Comment by bluerai** [comment]
Hi Michael, 

you are using an old version. Please read READ.ME and re-install from 

https://github.com/bluerai/ioBroker.mobile-alerts

On start of the adapter you have to see:

.. starting. Version   **_0.5.1_**   in /opt/iobroker/node_modules/iobroker.mobile-alerts,...

bluerai

**Comment by MichaelSchaaf999** [comment]
Dear bluerai,
yes - worked fine. Now we have the matching values from the system back in IOBroker.
Question: Not showing updates in IOBroker...  that would be more elegant if this would be the case, for sure.
If you cannot provide that: Is there another way to check the version and monitor it and tell the system owner that there is an update necessary? Just getting trapped here is not necessary, I think. And you will have less work yourself if the users have some kind of information on these updates. 
What do your think?
Best regards
Michael

**Comment by bluerai** [comment]
Hi Michael, thanks for your response note!

.. and about messaging the updates: you are right. This is truely an open issue and I will be working on that .. . give me some time :-)

For now I'm closing this issue.

bluerai

**Comment by MichaelSchaaf999** [comment]
Thanks, bluerai,
your roadmap is very much appreciated. Keep going.
Best regards
Michael

**Comment by pho59** [comment]
Hi bluerai,
at first many thanks that you provided this mobile-alerts Adapter.Had some issues to install your adapters via code directly. Tried a lot of possible ways, successful way was to unzip files into designated folder /opt/iobroker/node_modules/iobroker.mobile-alerts and changed ownership of these files to iobroker. But: after playing around with iobroker, the whole directory was either nearly emptied or even deleted. In some cases, the directory node_modules was copied into this folder. The only fix I found was to delete instance and Adapter and re-install from scratch. Any glue what happened?
